container-images/pyspark-slim/Dockerfile

```
@@ -33,27 +28,17 @@ ENV SPARK_HOME=/usr/local/spark
 ENV PATH="${PATH}:${SPARK_HOME}/bin"
 ENV SPARK_OPTS="--driver-java-options=-Xms1024M --driver-java-options=-Xmx4096M --driver-java-options=-Dlog4j.logLevel=info"
 
-COPY setup_spark.py /opt/setup-scripts/setup_spark.py
+COPY setup_spark.py /opt/setup-scripts/
+COPY requirements.txt /opt/setup-scripts/
```

Do a single layer COPY:
```suggestion
COPY setup_spark.py requirements.txt /opt/setup-scripts/
```